infra/fendermint/scripts/fendermint.toml

```diff
@@ -88,7 +89,8 @@ docker run \
   --env FM_VALIDATOR_KEY__PATH=/data/${NODE_NAME}/${VALIDATOR_PRIV_KEY_PATH} \
   --env FM_VALIDATOR_KEY__KIND=ethereum \
   --env TENDERMINT_RPC_URL=http://${CMT_CONTAINER_NAME}:26657 \
-  --env IROH_RPC_ADDR=${IROH_CONTAINER_NAME}:4919 \
+  --env IROH_PATH=/data/${NODE_NAME}/iroh-fendermint \
```

Comment:
What kind data will be stored in `IROH_PATH`? How large this folder is going to be? Does it thematically belong to the data or to the blocks? Many questions 🙂

Reply:
I think in your world this is "data" the actual LAAAARGE data you will be hopefully storing